src/Phpbb/TranslationValidator/Validator/FileListValidator.php

```diff
@@ -195,10 +204,19 @@ public function validate()
 					{
 						$level = Output::FATAL;
 					}
+					else if (in_array(substr($origin_file, -4), array('.gif', '.png')) && $direction === 'rtl')
```

Comment:
2 comments on this:

1. in_array is case-sensitive while extensions can be mixed case. Is this a problem that needs consideration?
2. I know this method wasn't added in or modified for this pr, but in `getFileList` instead of doing `str_replace` to remove the path to just have the file name, perhaps call `getFilename` instead of `getPathname`?

Reply:
@1.: I would say this was no problem in the past and maybe we can just add "only use lower case extensions" to the rules.
@2.: Maybe worth a new issue to keep track of this possible improvement?